### ISSUE TITLE
SQL Colour Coding Highlight on Stored Procedures

### ISSUE THREAD

**Issue by gordan-bobic** [shatteredsilicon/ssm-submodules#183]
It looks like CREATE PROCEDURE statements returned in QAN are not colour coded like regular statements and CREATE TABLE definitions.

Is there a way we can apply the highlight to them to make them easier to skim-read?

**Comment by oblitorum** [comment]
I notice that the highlighting doesn't always fail, for example

![2023-10-13 00_42_24-package json - qan-app  WSL_ Ubuntu-20 04  - Visual Studio Code](https://github.com/shatteredsilicon/ssm-submodules/assets/61085039/c8efbcc4-4859-4ec0-8baa-8cf3069e1139)

![2023-10-13 00_41_48-](https://github.com/shatteredsilicon/ssm-submodules/assets/61085039/68f254bf-bdd8-44ad-ada3-5e69368a0e1b)

One works but not the other, seems it fails on more complex CREATE PROCEDURE queries.

The highlighting is done with a npm package -> highlight.js, I changed the way the code gets highlighted, now the complex one also gets highlighted

![image](https://github.com/shatteredsilicon/ssm-submodules/assets/61085039/e8943ea8-a502-426d-9a85-d7ca1128c82a)

I think this will probably fix your case too, but I also notice that the "BEGIN" keyword is not highlighted like other keywords at above screenshot, it feels like the highlight feature this npm package provided is not very promising (at least at the version we used, which is 10.4.1)

**Comment by gordan-bobic** [comment]
Fair enough, if you can fix it easily, please try to upstream the patch. It isn't a massive problem that will prevent the next release. I'll relabel this as "upstream" in terms of release.
